packages/@aws-cdk/aws-kinesis/test/test.stream.ts

```diff
@@ -303,6 +304,22 @@ export = {
     test.done();
   },
 
+  'encryption key cannot be supplied with UNENCRYPTED as the encryption type'(test: Test) {
+    test.throws(() => {
```

Comment:
Can you scope the `throws` to just the thing that's supposed to raise?

```ts
    'encryption key cannot be supplied with UNENCRYPTED as the encryption type'(test: Test) {
      const stack = new Stack();
      const key = new kms.Key(stack, 'myKey');

      test.throws(() => {
        new Stream(stack, 'MyStream', {
          encryptionKey: key,
          encryption: StreamEncryption.UNENCRYPTED,
        });
      }, /encryptionKey is specified, so 'encryption' must be set to KMS/);

      test.done();
    },
```

Reply:
done